Dockerfile

```diff
@@ -11,40 +10,29 @@ RUN \
   apt-get -y upgrade && \
   apt-get -y install apt-transport-https ca-certificates curl
 
-# Install AppOptics Host Agent from their Ubuntu repo
+# Configure AppOptics Host Agent Ubuntu repo
 COPY ./conf/appoptics-xenial-repo.list /etc/apt/sources.list.d/appoptics-snap.list
 
 RUN \
   curl -L https://packagecloud.io/AppOptics/appoptics-snap/gpgkey | apt-key add - && \
   apt-get update && \
-  apt-get -y install appoptics-snaptel && \
+  apt-get -y install appoptics-snaptel=2.0.0-ao1.1475 && \
```

Comment:
why not moving this to an ENV var at the top ?

Reply:
Great point, will move this up top.